tikv-control.md

```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
 
 ### Compact data of each TiKV manually
 
-Use the `compact` command to manually compact data of each TiKV. If you specify the `--from` and `--to` options, then their flags are also in the form of escaped raw key.
+Use the `compact` command to manually compact data of each TiKV.
 
-- Use the `--host` option to specify the TiKV that needs to perform compaction.
-- Use the `-d` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
+- Use the `--from` and `--to` options to specify the compact range in the form of escaped raw key. If not set, the whole range will be compacted.
+- Use the `--region` option to compact range of a specific region. If set, `--from` and `--to` will be ignored.
+- Use the `--db` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
```

Comment:
Please add this to Chinese.  Thanks~

Reply:
https://github.com/pingcap/docs-cn/pull/11026